pod/main/static/js/videojs-info-controlbar.js

```diff
@@ -1,3 +1,72 @@
+// Esup-Pod videojs-info-controlbar
+
+// Read-only globals defined in video-script.html
+/*
+  global player
+*/
```

Comment:
- [Documentation en-tête d'un fichier](https://jsdoc.app/tags-file)
- [Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#file-headers)

On pourrait mettre un truc du style :
```js
/**
 * @file This file contains functions for managing video playback with Video.js.
 *  
 * Read-only globals defined in video-script.html
 */
```

Qu'en pense-tu ?
On pourrait aussi mettre la version du fichier `@since  x.x.x`, mais je n'ai pas cherché dans quelle version ça a été ajouté.

Reply:
En 2.0.0

Donc ça ferait :
```js
/**
 * @file This file contains functions for managing video playback with Video.js.
 * @since 2.0.0  
 *
 * Read-only globals defined in video-script.html
 */
```